src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -938,37 +938,40 @@ users:
-+
-Examples:
-....
-wget -P /opt/proactive/ http://192.168.99.1:8080/rest/node.jar
+- *kubernetesNamespace:* (Optional) Kubernetes namespace if different from _default_ namespace.
 
-wget -P /opt/proactive/ --no-check-certificate https://192.168.99.1:8443/rest/node.jar
-....
+- *cleaningDelaySeconds:* (Optional) Cleaning delay, in seconds until cleaning service starts. Default value is 120 seconds.
+
+- *cleaningFrequencySeconds:* (Optional) Cleaning frequency, Kubernetes node cleaning is done every x seconds. Default value is 30 seconds.
 
-- *rmCredentials:* path to a file which contains the credentials of an administrator user which will own the node source.
 - *scale:* Number of pods (ProActive Nodes) to create. This parameter is used when the <<../admin/ProActiveAdminGuide.html#_node_source_policies,Node Source Policy>> is not a <<../admin/ProActiveAdminGuide.html#_dynamic_policy,Dynamic Policy>>.
-- *javaNodeJarCommand:* command used to start the ProActive Node. The default value needs to be modified according to the location of the java command.
+
+- *image:* Docker image used to deploy ProActive Nodes. Example: java:8-jre
+
+- *rmHostname:* The hostname or the public IP address of the host operating the Resource Manager. This address needs to be accessible from the Kubernetes server. For example, `try.activeeon.com`.
+
+- *nodeJarURL:* The full URL path of the node.jar to download the ProActive node.jar on each new created pod. The URL needs to be accessible from the Kubernetes pods. For example, `try.activeeon.com/rest/node.jar`.
+
+- *jvmProperties:* Java options appended to the command used to start the node on the remote host. As the <<../admin/ProActiveAdminGuide.html#_proactive_message_routing,ProActive Message Routing>> is required by the Kubernetes Infrastructure, PAMR properties must be defined.
 +
 Example:
 ....
-java -jar /opt/proactive/node.jar
+-Dproactive.communication.protocol=pamr -Dproactive.useIPaddress=true -Dproactive.pamr.router.address=192.168.99.1
 ....
 
- - *jvmProperties:* Java options appended to the command used to start the node on the remote host. As the <<../admin/ProActiveAdminGuide.html#_proactive_message_routing,ProActive Message Routing>> is required by the Kubernetes Infrastructure, PAMR properties must be defined.
+- *startupScript:* VM startup script to launch the ProActive nodes. To run the ProActive nodes, this script is also expected to download and install the Java Runtime Environment if it is not already installed in the specified image, then download and and execute ProActive node.jar. The script can use <<_cloud_infrastructure_startup_script,some arguments>>. For example, it can use `%nodeJarUrl%` to represent the full URL path for downloading the _node.jar_ file. These arguments will be interpreted by Kubernetes Infrastructure later. This is an optional property. If left blank, a default script is automatically generated for the Linux OS.
```

Comment:
why not describing in a table all available arguments ?

Reply:
we don't have much variables. I don't see why putting them in a table but as u want

Reply:
well, how do you know what nodeNamingOption (see example below) means ?

Reply:
Edit, these patterns are already described in the referenced section <<_cloud_infrastructure_startup_script,some arguments>>